### PR TITLE
removed __api__ endpoint

### LINKED ISSUE #194: [Long Term Support] Remove /__api__ end point
It's currently broken and returning 500's

https://github.com/mozilla-services/absearch/blob/master/absearch/server.py#L259

[Part of the Long Term Support Plan](https://docs.google.com/document/d/1rWUq-CBuOYWJUz-qJohisX3gppz1rxBe6nD8iGHlOeU/edit#)

### PR DESCRIPTION
**Pull request by tiftran**
fixes #194

### CODE REVIEW

File: absearch/tests/test_server.py [hunk 1]
```diff
@@ -59,35 +59,6 @@ def test_version():
...
 
-
-def test_swagger():
-    app = get_app()
-    # test the APIs
-    res = app.get('/__api__')
-
-    # make sure it's compliant
-    swagger_dict = yaml.load(res.body, Loader=yaml.FullLoader)
-    parser = SwaggerParser(swagger_dict=swagger_dict)
```
Comment (leplatrem) [inline absearch/tests/test_server.py:70]: This swagger parser dependency should not be useful anymore

File: absearch/server.py [hunk 2]
```diff
@@ -256,17 +256,6 @@ def get_cohort_settings(**kw):
...
 
-@app.route('/__api__')
-def get_swagger(**kw):
-    host = request.headers.get('X-Forwarded-Host')
-    if host is None:
-        host = request.headers.get('Host', 'localhost')
-
-    scheme = request.headers.get('X-Forwarded-Proto', 'https')
-    options = {'HOST': host, 'VERSION': __version__, 'SCHEME': scheme}
-    return template('swagger', **options)
```
Comment (leplatrem) [inline absearch/server.py:267]: The `swagger.tpl` template should be useless now

**Review by leplatrem — APPROVED** [review]
r+wc

You should be able to remove more stuff related to swagger

File: absearch/tests/test_server.py (hunk 1 above)
Comment (tiftran) [reply absearch/tests/test_server.py:70]: will do

**Comment by tiftran** [comment]
gah, i guess i need to set up my user for signed and verified commits

**Pull request merged**